util/GenerateTLSCertificate.go

```diff
@@ -106,17 +106,7 @@ func GetIPv4ForInterfaceName(ifname string) (ip string) {
 }
 
 func main() {
-	fmt.Printf("Enter interface name: ")
-	input = bufio.NewReader(os.Stdin)
-	var ifName []byte
-	for {
-		data, prefix, _ := input.ReadLine()
-		ifName = append(ifName, data...)
-		if !prefix {
-			break
-		}
-	}
-
+	ifName := os.Args[1]
```

Comment:
Since interface is passed as cli argument, I think we don't need 'input' variable.